packages/documentation/docs/integration/flow-of-funds.md

```diff
@@ -0,0 +1,119 @@
+# Flow of funds
+
+Rafiki communicates with the Account Servicing Entity over webhook events, which are defined in the _Getting Started_ section. Each event requires the Account Servicing Entity to either deposit or withdraw liquidity into or from Rafiki. This document will provide an example flow for each of the defines webhook events.
```

Comment:
```suggestion
Rafiki communicates with the Account Servicing Entity over webhook events, which are defined in the _Getting Started_ section. Most events require the Account Servicing Entity to either deposit or withdraw liquidity into or from Rafiki. This document will provide an example flow for each of the defines webhook events.
```
Not all events: `incoming_payment.created` (and by extension of events, the `payment_pointer.not_found` one